airbyte-integrations/connectors/source-sendgrid/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_sendgrid"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "==0.51.39"
+airbyte-cdk = "==0.65.0"
 requests = "==2.31.0"
 pandas = "==2.1.1"
```

Comment:
Nit: Same with other deps - should we pin minor version but unpin the patch version?